workshop_app/test_models.py

```diff
@@ -0,0 +1,190 @@
+			department='IT', institute='iit', phone_number='9930011492',
+			is_email_verified=1)
+
+	wt1 = WorkshopType.objects.create(workshoptype_name='ISCP',
+			workshoptype_description='Introduction to Scientific Computing in\
+			Python <br> > Numpy <br> > Matplotlib <br> > iPython <br>',
+			workshoptype_duration='1day, 8hours a day')
+
+	rw = RequestedWorkshop.objects.create(
+						requested_workshop_instructor=testUser2,
+						requested_workshop_coordinator=demoUser2,
+						requested_workshop_title=wt1,
+						requested_workshop_date='2017-07-24'
+						)
+	
+	pw = ProposeWorkshopDate.objects.create(
+					proposed_workshop_coordinator=demoUser2,
+					proposed_workshop_instructor=testUser2,
+					proposed_workshop_title=wt1,
+					proposed_workshop_date='2017-07-06',
+					condition_one=1,
+					condition_two=1,
+					condition_three=1
+					)
+
+
+def tearDownModule():
+    User.objects.all().delete()
+    Profile.objects.all().delete()
+    ProposeWorkshopDate.objects.all().delete()
+    RequestedWorkshop.objects.all().delete()
+    WorkshopType.objects.all().delete()
+
+class ProfileModelTest(TestCase):
+	'''	
+		This class tests the Profile Model
+	'''
+	def setUp(self):
+		'''
+			ip is for instructor profile and cp is for coordinator profile
+		'''	
+		self.testuser1 = User.objects.get(username='testuser1')
+		self.demouser1 = User.objects.get(username='demouser1')
+		
+
+		self.ip1 = Profile.objects.create(user=self.testuser1, position='instructor',
+			department='cs', institute='ace', phone_number='9930011492',
+			is_email_verified=1)
+
+		self.cp1 = Profile.objects.create(user=self.demouser1, position='coordinator',
+			department='IT', institute='iit', phone_number='9930011492',
+			is_email_verified=1)
+
+
+	def test_profile_model(self):
+		self.assertEqual(self.demouser1.email,'doke.akshen@gmail.com')
+		self.assertEqual(self.testuser1.email,'doke.akshen@gmail.com')
+		self.assertEqual(self.ip1.position,'instructor')
+		self.assertEqual(self.cp1.position,'coordinator')
+
+
+class WorkshopTypeModelTest(TestCase):
+	'''
+		This class tests the WorkshopType Model
+	'''
+
+	def setUp(self):
+		self.wt1 = WorkshopType.objects.create(workshoptype_name='ISCP',
+			workshoptype_description='Introduction to Scientific Computing in\
+			Python <br> > Numpy <br> > Matplotlib <br> > iPython <br>',
+			workshoptype_duration='1day, 8hours a day')
+
+		self.wt2 = WorkshopType.objects.create(workshoptype_name='Basic Python',
+			workshoptype_description='Basic Python <br> > DataTypes <br> \
+			> Conditions <br> > Loops <br> > Functions'
+			,workshoptype_duration='3days, 8hours a day')
+
+	def test_workshoptype_model(self):
+		self.assertEqual(self.wt2.workshoptype_duration,'3days, 8hours a day')
+		self.assertEqual(self.wt1.workshoptype_name, 'ISCP')
+
+
+class WorkshopTest(TestCase):
+	'''
+		This class tests the Workshop Model
+	'''
+
+	def setUp(self):
+		'''
+		 w is workshop object
```

Comment:
Docstrings are used for expaining what a function, method, module or class is about. This is bad usage of docstrings.